src/main/java/org/asciidoctor/asciidoclet/LazyDocCommentTableProcessor.java

```diff
@@ -26,40 +26,46 @@
 
 class LazyDocCommentTableProcessor {
 
-    @SuppressWarnings("unchecked")
-    static void processComments(DocCommentTable table, Function<Comment, Comment> mapper) {
-        // Use heckin' raw-types because LazyDocCommentTable.Entry has private access, so we
-        // cannot statically express its type here.
-        Map map;
-        Function<Object, Object> converter;
-        try {
-            Field tableField = LazyDocCommentTable.class.getDeclaredField("table");
```

Comment:
To get some performance, we could initialize `tableField` and `ctor` in a static block like seen below. If there's any issue with the reflection part, better to fail asap.

```
class LazyDocCommentTableProcessor {

    static Field tableField;

    static {
        try {
            tableField = LazyDocCommentTable.class.getDeclaredField("table");
            tableField.setAccessible(true);
        } catch (NoSuchFieldException e) {
            throw new RuntimeException(e);
        }
    }
```

Reply:
Okay, let me try.

Reply:
I noticed that it would be even better to make the functions to extract comment field value and to construct a comment object as you see in the change.